### PR TITLE
[67] Style the Product Details Using CSS-Frontend

### PR DESCRIPTION
**Pull request by gracioust**
## Task: `Style the Product Details Using CSS-Frontend`
- `Styled the Product Details Using CSS-Frontend`
product1
product2
product3
product4
product5
product6

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @gracioust 

But, the last two screenshots don't look too ok. The cards look compressed. Kindly rework them

**Comment by gracioust** [comment]
Reworked the screenshots
product1
product2
product3
product4
product5
product6

**Comment by gracioust** [comment]
> Well done @gracioust
> 
> But, the last two screenshots don't look too ok. The cards look compressed. Kindly rework them

Thanks, I have made the changes.

**Comment by adaugochi** [comment]
well done @gracioust 
we are making progress
Kindly make more adjustment. there are spaces around it, let it fit properly

File: submissions/tosinIshola/HTML/Home.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<html lang="en">
```
Comment (adaugochi) [inline submissions/tosinIshola/HTML/Home.html:1]: what is this home.html for?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by gracioust** [comment]
I will create another PR

**Pull request closed without merging**